superset-frontend/src/dashboard/actions/dashboardState.js

```diff
@@ -581,7 +581,7 @@ export function addSliceToDashboard(id, component) {
     ]).then(() => {
       dispatch(addSlice(selectedSlice));
 
-      if (selectedSlice && selectedSlice.viz_type === 'filter_box') {
+      if (selectedSlice) {
```

Comment:
If the code in the `if` block was meant to run only for filter box, shouldn’t we remove it?

Reply:
Great call! It by removing this I was able to remove even more code!